presto-hive/src/main/java/io/prestosql/plugin/hive/HiveWriterFactory.java

```diff
@@ -549,6 +549,37 @@ else if (insertExistingPartitionsBehavior == InsertExistingPartitionsBehavior.ER
                 hiveWriterStats);
     }
 
+    private String computeFileName(OptionalInt bucketNumber)
+    {
+        if (isCreateTransactionalTable) {
+            // Hive is very opinionated on naming pattern for "original" files of transactional tables.
```

Comment:
avoid being pejorative, remove "very opinionated".

Reply:
`opinionated` is not pejorative. Is it? I can remove "very". I need to keep opinionated (or replace it with sth); otherwise sentence does not parse :)

Reply:
"opinionated" can be received as pejorative. You can say "Hive expects/requires file names to conform to the following naming pattern ..."

Reply:
Sure

Reply:
Add this comment here too: https://github.com/prestosql/presto/pull/2930#discussion_r456398810
(ie this code place, can still be in your PR)

Reply:
You mean to put information that hive ~is opinionated~ requires specific naming of original files in `BackgroundHiveSplitLoader` around the code which handles original files?

Reply:
i guess it needs to be in both places. Here, because we produce new file name and we apply some logic where table is CTAS.
in BHSL too, because we fail if an original file doesn't meet our expectations. I added it there (in one place only...)